openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx

```diff
@@ -145,6 +149,23 @@ const DatabaseSchemaPage: FunctionComponent = () => {
   const handleShowDeletedTables = (value: boolean) => {
     setShowDeletedTables(value);
     handlePageChange(INITIAL_PAGING_VALUE);
+
+    const path = location.pathname;
+    const searchParams = new URLSearchParams(location.search);
+
+    searchParams.set('showDeletedTables', value.toString());
```

Comment:
Let's revert this

Reply:
done, updated this such that it uses useTableFilter hook